compiler+runtime/CMakeLists.txt

```diff
@@ -192,7 +192,7 @@ separate_arguments(clang_system_include_dirs)
 
 set(clang_system_include_flags "")
 foreach(dir ${clang_system_include_dirs})
-  list(APPEND clang_system_include_flags -I${dir})
+  list(APPEND clang_system_include_flags ${dir})
```

Comment:
On my system, the output from the command above already includes `-I` prefixes. This results in strings like `-I-I/include/path` being passed to the compiler. Please double check.